packages/dx-react-grid-bootstrap3/src/templates/layout.jsx

```diff
@@ -12,36 +13,55 @@ Root.propTypes = {
     PropTypes.node,
     PropTypes.arrayOf(PropTypes.node),
   ]),
+  className: PropTypes.string,
 };
 
 Root.defaultProps = {
   children: undefined,
+  className: undefined,
 };
 
-export const Header = ({ children }) =>
-  children && <div className="panel-heading" style={{ paddingBottom: '5px' }}>{children}</div>;
+export const Header = ({
+  children, className,
+  style, ...restProps
+}) => !!children && (
+  <div
+    className={classNames('panel-heading', className)}
+    style={{ paddingBottom: '5px', ...style }}
+    {...restProps}
+  >
+    {children}
+  </div>
+);
 
 Header.propTypes = {
   children: PropTypes.oneOfType([
     PropTypes.node,
     PropTypes.arrayOf(PropTypes.node),
   ]),
+  className: PropTypes.string,
+  style: PropTypes.object,
 };
 
 Header.defaultProps = {
   children: undefined,
+  className: undefined,
+  style: null,
 };
 
-export const Footer = ({ children }) =>
-  children && <div className="panel-footer">{children}</div>;
+export const Footer = ({ children, className, ...restProps }) =>
+  !!children &&
```

Comment:
Why `!!`?

Reply:
@SergeyAlexeev  Because if don't pass children to the FooterPlaceholder my application broken with an exception.